common/src/main/res/values-bg/strings-common.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2022 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+<resources>
+    <string name="common_Today">Днес</string>
+    <string name="common_Yesterday">Вчера</string>
+    <string name="common_DaysAgo">Преди %1$d дни</string>
+    <string name="common_PastWeek">Последната седмица</string>
+    <string name="common_PastMonth">Последния месец</string>
+    <string name="common_MinutesAgo">преди %1$dмин.</string>
+    <string name="common_HoursAgo">преди %1$dч.</string>
+    <string name="common_JustNow">Току-що</string>
+    <string name="common_hour_abbreviation">ч.</string>
+    <string name="common_min_abbreviation">мин.</string>
+    <string name="common_seconds_abbreviation">сек.</string>
```

Comment:
Nit: We should probably name these consistently (Camel-case or underscores)

Reply:
Makes sense, I'll probably create a separate task to align that. Because we'll need to re-sync with smartling 🤦